src/sunmatrix/cusparse/sunmatrix_cusparse.cu

```diff
@@ -43,6 +43,12 @@ static SUNMatrix SUNMatrix_cuSparse_NewEmpty(SUNContext sunctx);
 static cusparseStatus_t CreateSpMatDescr(SUNMatrix, cusparseSpMatDescr_t*);
 #endif
 
+#if CUDART_VERSION >= 12000
+#define SPMV_ALG CUSPARSE_SPMV_CSR_ALG1
+#else
+#define SPMV_ALG CUSPARSE_MV_ALG_DEFAULT
+#endif
```

Comment:
I'm glad it was as simple as this (though finding `CUSPARSE_SPMV_CSR_ALG1` may not have been trivial)